.travis.yml

```diff
@@ -24,9 +23,9 @@ addons:
 env:
     global:
         # The following versions are the 'default' for tests, unless
-        # overidden underneath. They are defined here in order to save having
+        # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
-        - NUMPY_VERSION=1.9
+        - NUMPY_VERSION=stable
```

Comment:
The whole build -- https://travis-ci.org/ejeschke/ginga/builds/121235602

One of the jobs -- https://travis-ci.org/ejeschke/ginga/jobs/121235604